tests/writeConcern/writeconcern-001.phpt

```diff
@@ -6,25 +6,26 @@ MongoDB\Driver\WriteConcern construction
 <?php
 require_once __DIR__ . "/../utils/basic.inc";
 
-$w = new MongoDB\Driver\WriteConcern(MongoDB\Driver\WriteConcern::MAJORITY);
-$w2 = new MongoDB\Driver\WriteConcern(MongoDB\Driver\WriteConcern::MAJORITY, 1000);
+var_dump(new MongoDB\Driver\WriteConcern(MongoDB\Driver\WriteConcern::MAJORITY));
+var_dump(new MongoDB\Driver\WriteConcern(MongoDB\Driver\WriteConcern::MAJORITY, 1000));
 
-$w3 = new MongoDB\Driver\WriteConcern(2);
-$w4 = new MongoDB\Driver\WriteConcern(2, 2000);
+var_dump(new MongoDB\Driver\WriteConcern(2));
+var_dump(new MongoDB\Driver\WriteConcern(2, 2000));
 
-$w5 = new MongoDB\Driver\WriteConcern("tagname");
-$w6 = new MongoDB\Driver\WriteConcern("string", 3000);
+var_dump(new MongoDB\Driver\WriteConcern("tagname"));
+var_dump(new MongoDB\Driver\WriteConcern("string", 3000));
 
-$w7 = new MongoDB\Driver\WriteConcern("string", 3000, false, false);
-$w8 = new MongoDB\Driver\WriteConcern("string", 3000, true, true);
+var_dump(new MongoDB\Driver\WriteConcern("string", 4000, true));
```

Comment:
I changed the test to use incrementing values for `$wtimeout` to make the test diffs easier to follow (on failure).